src/CairoMakie.jl

```diff
@@ -531,15 +531,41 @@ function draw_plot(scene::Scene, screen::CairoScreen, primitive::Combined)
 end
 
 function draw_plot(screen::CairoScreen, scene::Scene)
+
+    # get the parent area to correct for its pixel size when translating
+    parent_area = if isnothing(scene.parent)
+        pixelarea(scene)[]
+    else
+        pixelarea(scene.parent)[]
+    end
+    parent_area_height = widths(parent_area)[2]
```

Comment:
```suggestion
parent_area_height = round(Int, screen.surface.height)

Reply:
why should the parent always be the screen? if we have multiple nested scenes?

Reply:
the screen is always global and toplevel, right?  The solution you posted also works, this is just an alternative.